R/a11y_panel.R

```diff
@@ -0,0 +1,250 @@
+            "This website is partially compliant with the ",
+            external_link(
+              href = "https://www.w3.org/TR/WCAG22/",
+              "Web Content Accessibility Guidelines version 2.2 AA standard"
+            ),
+            " due to the non-compliances listed below."
+          ),
+          shiny::tags$h3("Non accessible content"),
+          shiny::tags$p("The content listed below is non-accessible for the following reasons.
+             We will address these issues to ensure our content is accessible."),
+          shiny::tags$div(tags$ul(
+            tagList(lapply(specific_issues, shiny::tags$li))
+          ))
+        )
+      },
+      shiny::tags$h3("Disproportionate burden"),
+      shiny::tags$p("Not applicable."),
+      shiny::tags$h2("How we tested this website"),
+      shiny::tags$p(
+        "The template used for this website was last tested on",
+        date_template_reviewed,
+        " against ",
+        external_link(
+          href = "https://www.w3.org/TR/WCAG22/",
+          "Accessibility Guidelines WCAG2.2"
+        ),
+        ". The test was carried out by the ",
+        external_link(
+          href = "https://digitalaccessibilitycentre.org/",
+          "Digital accessibility centre (DAC)"
+        ),
+        "."
+      ),
+      shiny::tags$p(
+        "DAC tested a sample of pages to cover the core functionality of the service including:"
+      ),
+      shiny::tags$div(tags$ul(
+        shiny::tags$li("navigation"),
+        shiny::tags$li("interactive dropdown selections"),
+        shiny::tags$li("charts, maps, and tables")
+      )),
+      shiny::tags$p(
+        "This specific website was was last tested on ", date_tested, " against ",
+        external_link(
+          href = "https://www.w3.org/TR/WCAG22/",
+          "Accessibility Guidelines WCAG2.2"
+        ),
+        ". The test was carried out by the ",
+        external_link(
+          href = "https://www.gov.uk/government/organisations/department-for-education",
+          "Department for Education (DfE)"
+        ),
+        "."
+      ),
+      shiny::tags$h2("What we're doing to improve accessibility"),
+      shiny::tags$p(
+        "We plan to continually test the service for accessibility issues, and are working ",
+        "through a prioritised list of issues to resolve."
+      ),
+      shiny::tags$p(
+        if (!is.na(repo_url)) {
+          shiny::tagList(
+            "Our current list of issues to be resolved is available on our ",
+            external_link(
+              href = paste0(repo_url, "/issues"),
+              "GitHub issues page"
+            ),
+            "."
+          )
+        } else {
+          " "
+        },
+      ),
+      shiny::tags$h2("Preparation of this accessibility statement"),
+      shiny::tags$p(
+        paste0(
+          "This statement was prepared on ",
+          date_prepared,
+          " and last reviewed on ",
+          date_reviewed,
```

Comment:
Do we need another test to make sure date reviewed is the same or later as date prepared? 

![image](https://github.com/user-attachments/assets/63dc26b7-076c-481e-a61c-58a1a6c8955f)

Reply:
There is a scenario where we do a reaudit and update the default date to be the latest audit, and that is later than the last time the dashboard itself was reviewed. However, I think if we make it explicit in the error message that that just means we need to re-review their dashboard it will be an excellent way to catch dashboards we need to rereview after our central stuff has been updated / reaudited!